main.go

```diff
@@ -1,31 +1,76 @@
 package main
 
 import (
+	"database/sql"
+	"fmt"
 	"github.com/gin-gonic/gin"
-	"net/http"
+	_ "github.com/lib/pq"
+	"log"
+)
+
+const (
+	DB_USER     = "twvjbojy"
+	DB_PASSWORD = "H0kJZUiDgpgweAWNWdLRgI6v6-IEyxyr"
+	DB_NAME     = "twvjbojy host=john.db.elephantsql.com"
+	DB_HOST     = "john.db.elephantsql.com"
 )
 
 var router *gin.Engine
+var db *sql.DB
 
-func main() {
-	router = gin.Default()
-	initializeRoutes()
-	router.Run()	// Runs on default port 8080
+type patient struct {
+	ID         int    `json:"id"`
+	Name       string `json:"name"`
+	Illness    string `json:"illness"`
+	BirthDate  string `json:"birth_date"`
+	LocationId int    `json:"location_id"`
 }
 
-// This function will initialize all the routes
-func initializeRoutes() {
-	router.GET("/hello", helloHandler)
-	router.GET("/greet/:name", greetHandler)
+type location struct {
+	ID         int    `json:"id"`
+	Name       string `json:"name"`
+	HospitalId int    `json:"hospital_id"`
 }
 
-// Returns a HelloWorld json message
-func helloHandler(c *gin.Context) {
-	c.JSON(http.StatusOK, gin.H{ "message": "Hello World!"})
+type hospital struct {
+	ID         int    `json:"id"`
+	Name       string `json:"name"`
+	HospitalId int    `json:"max_patient_amount"`
```

Comment:
This attribute name doesn't seem to match. Also, add the database representation / name for all the attributes. 
```suggestion
	MaxPatientAmount int    `json:"max_patient_amount" db:"max_patient_amount"`
```

Reply:
Sorry, corrected it :)